app/src/main/res/layout/activity_zim_host.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayoutId"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
+    tools:context=".webserver.ZimHostActivity"
+    >
+  <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/appBarLayout2"
```

Comment:
appBarLayout2? you can just call it appBarLayout or if you need to distnguish it `zimHostAppBarLayout`

Reply:
Fixed. ebeff0abeb5cf364c2bba1d2b0413733ccc77711